typescript/packages/reality-data-client/package.json

```diff
@@ -67,7 +67,7 @@
     "eslint-plugin-deprecation": "^1.3.3",
     "rimraf": "^4.4.1",
     "source-map-loader": "^1.0.0",
-    "typescript": "~4.6.4",
+    "typescript": "~5.1.0",
```

Comment:
A major typescript version update might require a version bump. I'm thinking1.2.2.
Can you do a version bump with beachball?